countMVS.py

```diff
@@ -483,16 +508,28 @@ def is_console():
                 api_response = requests.get(check_url, headers=JSON_HEADER, auth=auth, verify=False)
             elif use_token:
                 api_response = requests.get(check_url, headers=json_token_header, verify=False)
-            if api_response.status_code == 401:
-                unauth_msg = "API call returned 401 Unauthorized."  # pylint: disable=invalid-name
-                if "locked out" in api_response.text:
-                    unauth_msg += "\nYour host has been locked out due to too many failed login attempts. " \
-                        "Please try again later."
-                elif use_password:
-                    unauth_msg += "\nYou have provided the incorrect password. Please rerun the script and try again."
-                elif use_token:
-                    unauth_msg += "\nYou have provided the incorrect token. Please rerun the script and try again."
-                sys.exit(unauth_msg)
+
+            if not api_response.ok:
+                if api_response.status_code == 401:
+                    unauth_msg = "API call returned 401 Unauthorized."  # pylint: disable=invalid-name
+                    if "locked out" in api_response.text:
+                        unauth_msg += "\nYour host has been locked out due to too many failed login attempts. " \
+                            "Please try again later."
+                    elif use_password:
+                        unauth_msg += "\nYou have provided the incorrect password. " \
+                            "Please rerun the script and try again."
+                    elif use_token:
+                        unauth_msg += "\nYou have provided the incorrect token. Please rerun the script and try again."
+                    sys.exit(unauth_msg)
+                elif api_response.status_code == 403 and use_token:
+                    sys.exit("API call returned 403 Forbidden. \nThe token provided has incorrect permissions. " \
+                        "Please rerun the script and try again")
+                else:
+                    if api_response.text:
+                        logging.debug("Error executing API call {}".format(api_response.text))
+                    else:
+                        logging.debug("Error executing API call empty api_response.text")
+                    sys.exit("Error connecting to API")
```

Comment:
This looks very similar to another code block above. Ideally it should be refactored to remove the duplication, but I don't think it's worth doing here because Smokey is making major changes under another ticket.